src/shadowbox/server/manager_metrics.ts

```diff
@@ -15,8 +15,40 @@
 import {PrometheusClient} from '../infrastructure/prometheus_scraper';
 import {DataUsageByUser, DataUsageTimeframe} from '../model/metrics';
 
+interface ServerMetricsTimeframe {
+  hours: number;
+}
+
+interface Duration {
+  seconds: number;
+}
```

Comment:
I think we should merge these two durations, to be consistent and avoid confusion.

If anything, Tunnel Time would need a more coarse resolution than the time frame, since it accumulates overlapping durations (one per user), while the time frame doesn't. We can still stick to seconds for Tunnel Time, since it would take 1 trillion users 24/7 for a year to reach the limits of 64 bits: log(24*3600*365*1,000,000,000,000)/log(2) = 64.7